### PR TITLE
Doc 1.1.9 - part 1

### PR DESCRIPTION
**Pull request by florentlb**
### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: documentation/src/main/antora/modules/ROOT/pages/component-configuration.adoc [hunk 1]
```diff
@@ -252,6 +202,64 @@ A common use of the ActiveIf condition consists in testing if a target property
...
+    // the generated layout put one configuration entry per line,
+    // customize it as much as needed
+    @GridLayout.Row({ "checkbox1" }),
+    @GridLayout.Row({ "checkbox2" }),
+    @GridLayout.Row({ "checkbox3" }),
+    @GridLayout.Row({ "configuration4" }),
+    @GridLayout.Row({ "configuration5" }),
+    @GridLayout.Row({ "configuration6" })
+})
+@Documentation("TODO fill the documentation for this configuration")
```
[the remark this answers is not in the document]
Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/component-configuration.adoc]: ok

File: documentation/src/main/antora/modules/ROOT/pages/build-tools-maven.adoc [hunk 2]
```diff
@@ -443,7 +443,7 @@ For this command to work, you need to register the plugin as follows:
 
 === Generating the component archive
 
-Component ARchive (*.car*) is the way to bundle a component to share it in the Talend ecosystem. It is a plain Java ARchive (*.jar*) containing a metadata file and a nested Maven repository containing the component and its depenencies.
+Component ARchive (*.car*) is the way to bundle a component to share it in the Talend ecosystem. It is a plain Java ARchive (*.jar*) containing a metadata file and a nested Maven repository containing the component and its dependencies.
```
[the remark this answers is not in the document]
Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/build-tools-maven.adoc]: ok

File: documentation/src/main/antora/modules/ROOT/pages/testing-http.adoc [hunk 3]
```diff
@@ -150,6 +150,8 @@ You can create it manually, as described in <<component-runtime-http-junit>>, bu
 
 In most cases, use `src/test/resources`. If `new File("src/test/resources")` resolves the valid folder when executing your test (Maven default), then you can just set the system property to `true`. Otherwise, you need to adjust accordingly the system property value.
 
+NOTE: When set to `false`, the capture is not disabled. Instead, captures are saved in a `false/` directory.
```
[the remark this answers is not in the document]
Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/testing-http.adoc]: ok

File: documentation/src/main/antora/modules/ROOT/pages/creating-dataset-datastore.adoc [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+[[dataset-and-datastore]]
+
+Datasets and datastores are configuration types that define how and where to pull the data from. They are used at design time to create shared configurations that can be stored and used at runtime.
+
+All input and output components created using Talend Component Kit must reference a valid dataset. Each dataset must reference a datastore.
+
+- Datastore: The data you need to connect to the backend.
+- Dataset: A datastore coupled with the data you need to execute an action.
+
+image::datastore_and_dataset_validation.png[Dataset validation,window="_blank",link="_images/datastore_and_dataset_validation.png",50%]
```
[the remark this answers is not in the document]
Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/creating-dataset-datastore.adoc]: no as most links of the PDF, not only for images

**Comment by florentlb** [comment]
changes done

File: documentation/src/main/antora/modules/ROOT/pages/creating-dataset-datastore.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (rmannibucau) [reply documentation/src/main/antora/modules/ROOT/pages/creating-dataset-datastore.adoc]: So ley's use absolute links

File: documentation/src/main/antora/modules/ROOT/pages/component-configuration.adoc [hunk 5]
```diff
@@ -252,6 +202,64 @@ A common use of the ActiveIf condition consists in testing if a target property
...
+    @Option
+    @Documentation("")
+    private boolean checkbox1;
+
+    @Option
+    @Documentation("")
+    private boolean checkbox2;
+
+    @Option
+    @Documentation("")
```
Comment (rmannibucau) [inline documentation/src/main/antora/modules/ROOT/pages/component-configuration.adoc]: Not a todo but not a doc sample, maybe put a word on the reason of that option

**Pull request merged**